cmd/bacalhau/docker_run.go

```diff
@@ -103,38 +123,20 @@ var dockerRunCmd = &cobra.Command{
 			return err
 		}
 
-		shells := strings.Split(`/bin/sh
-/bin/bash
-/usr/bin/bash
-/bin/rbash
-/usr/bin/rbash
-/usr/bin/sh
-/bin/dash
-/usr/bin/dash
-/usr/bin/tmux
-/usr/bin/screen
-/bin/zsh
-/usr/bin/zsh`, "/n")
-
-		containsGlob := false
-		for _, entrypointArg := range jobEntrypoint {
-			if strings.ContainsAny(entrypointArg, "*") {
-				containsGlob = true
-			}
+		for _, i := range jobInputs {
+			jobInputVolumes = append(jobInputVolumes, fmt.Sprintf("%s:/inputs", i))
```

Comment:
If user specifies this multiple times will it clobber /inputs mountpoint and error?